posthog/event_usage.py

```diff
@@ -144,17 +162,48 @@ def report_bulk_invited(
             "current_member_count": current_member_count,
             "email_available": email_available,
         },
+        groups=groups(user.current_organization, user.current_team),
     )
 
 
-def report_org_usage(distinct_id: str, properties: Dict[str, Any]) -> None:
+def report_org_usage(organization_id: str, distinct_id: str, properties: Dict[str, Any]) -> None:
     """
     Triggered daily by Celery scheduler.
     """
     posthoganalytics.capture(
-        distinct_id, "organization usage report", properties,
+        distinct_id,
+        "organization usage report",
+        properties,
+        groups={"organization": organization_id, "instance": SITE_URL},
+    )
+    posthoganalytics.group_identify("organization", organization_id, properties)
+
+
+def report_org_usage_failure(organization_id: str, distinct_id: str, err: str) -> None:
+    posthoganalytics.capture(
+        distinct_id,
+        "organization usage report failure",
+        properties={"error": err,},
+        groups={"organization": organization_id, "instance": SITE_URL},
```

Comment:
Curious why we don't use `groups` here, but in any case probably better to use `report_user_action` everywhere.

Reply:
We don't have easy access to the organization object here, hence working with the id.

In this case `project` is also not relevant for this event - there's no sane value if you have multiple.